docs/developing/android.md

```diff
@@ -11,218 +11,28 @@ sidebar_label: Developing for Android
   />
 </head>
 
-This guide covers how to run and debug Ionic apps on Android emulators and devices using [Capacitor](../reference/glossary.md#capacitor) or [Cordova](../reference/glossary.md#cordova). Android apps can be developed on Windows, macOS, and Linux.
+import DocsCard from '@components/global/DocsCard';
+import DocsCards from '@components/global/DocsCards';
 
-## Android Studio
+:::info Looking for the legacy Android guide?
 
-[Android Studio](https://developer.android.com/studio/) is the IDE for creating native Android apps. It includes the [Android SDK](../reference/glossary.md#android-sdk), which will need to be configured for use in the command line.
+The Developing for Android guide has officially migrated to the Capacitor docs: https://capacitorjs.com/docs/android. Looking for the legacy docs? [Click here](/docs/v6/developing/android).
```

Comment:
- This link, `https://capacitorjs.com/docs/android`, should be inline to be accessible compliant.
- "Click here" also is too vague to be accessible compliant.

This [article](https://business.scope.org.uk/article/how-to-write-better-link-text-for-accessibility) is a good read to understand why links should be set in a certain way for web accessibility.

Reply:
Let me know your thoughts on the changes 👍